README.md

```diff
@@ -67,14 +67,15 @@ app = Starlite(route_handlers=[hello_world])
 - Layered Parameter declaration
 - Life Cycle Hooks
 - OpenAPI 3.1 schema generation
+- Piccolo ORM Support (via plugin)
```

Comment:
We do actually have a Piccolo ORM plugin though..?

[starlite/plugins/piccolo_orm.py](https://github.com/starlite-api/starlite/blob/main/starlite/plugins/piccolo_orm.py)